### PR TITLE
No more voms

### PR DESCRIPTION
**Pull request by alvarolopez**
# Description

Deprecate static mappings, and use OpenStack metadata for that.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] This change is a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details for
your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: doc/source/configuration.rst [hunk 1]
```diff
@@ -66,6 +78,37 @@ that you are applying the correct changes as listed in the following table.
...
+Setting VO mapping
+------------------
+
+In order to publish correct accounting records, ``cASO`` needs to know the VO that
+should be used to publish the records from a given project. In order to do so, you need
+to specify the correct mapping in each of the projects properties. The name of the
+property that will be used is defined in the ``vo_property`` configuration option, and
+defaults to ``accounting:VO``, therefore you can configure it as follows::
+
+     openstack project set --property acconting:VO=<VO FQAN> <project id>
```
Comment (enolfc) [inline doc/source/configuration.rst:110]:
```suggestion
     openstack project set --property accounting:VO=<VO FQAN> <project id>
```

**Comment by alvarolopez** [comment]
@enolfc are documentation and release notes enough?

File: doc/source/configuration.rst [hunk 2]
```diff
@@ -34,20 +34,32 @@ Otherwise, ``cASO`` will not be able to get the usages and will fail.
...
+``all``::
+
+    openstack role add --system all --user accounting reader
 
 Policy modifications
---------------------
-The accounting user needs access to Keystone so as to extract the users
-information. In this case, we can can grant the user just the rights for
-listing the users adding the appropriate rules in your policy configuration.
+------------------------
```
Comment (enolfc) [inline doc/source/configuration.rst:48]: this one is a bit confusing, I understand that with current OpenStack versions, what is needed is that `reader` role is added to the user that runs cASO, and no further changes are needed (so this whole Policy modifications section would be useless). Is this correct? 

If so, then I would adjust the previous section, by removing the 
```
In order to do so, we are going to setup a new role ``accounting`` a new user
``accounting``, adding it to each of the projects with that role::
```

And then moving this "Policy modifications" to something like "Using a dedicated role" or similar

File: doc/source/configuration.rst [hunk 3]
```diff
@@ -66,6 +78,37 @@ that you are applying the correct changes as listed in the following table.
...
+* Using the ``projects`` list in the ``[DEFAULT]`` section of your configuration file
+  (see below).
+
+Setting VO mapping
+------------------
+
+In order to publish correct accounting records, ``cASO`` needs to know the VO that
+should be used to publish the records from a given project. In order to do so, you need
+to specify the correct mapping in each of the projects properties. The name of the
+property that will be used is defined in the ``vo_property`` configuration option, and
```
Comment (enolfc) [inline doc/source/configuration.rst:107]: `vo_property` is not in the list of options below

**Comment by enolfc** [comment]
> @enolfc are documentation and release notes enough?
> 
> * https://caso.readthedocs.io/en/4.1.0.rc1/release-notes.html#rc1
> * https://caso.readthedocs.io/en/4.1.0.rc1/configuration.html#selecting-projects-to-get-usages

Added a couple of comments ;)

**Comment by alvarolopez** [comment]
> > @enolfc are documentation and release notes enough?
> > 
> > * https://caso.readthedocs.io/en/4.1.0.rc1/release-notes.html#rc1
> > * https://caso.readthedocs.io/en/4.1.0.rc1/configuration.html#selecting-projects-to-get-usages
> 
> Added a couple of comments ;)

Can you push some changes with that? I am a bit biased :-)

**Comment by alvarolopez** [comment]
> > @enolfc are documentation and release notes enough?
> > 
> > * https://caso.readthedocs.io/en/4.1.0.rc1/release-notes.html#rc1
> > * https://caso.readthedocs.io/en/4.1.0.rc1/configuration.html#selecting-projects-to-get-usages
> 
> Added a couple of comments ;)

Done.

**Review by enolfc — APPROVED** [review]
LGTM! Thanks

**Pull request merged**